scripts/rollup/build-all-release-channels.js

```diff
@@ -168,7 +168,10 @@ function processStable(buildDir) {
       const stats = fs.statSync(filePath);
       if (!stats.isDirectory()) {
         hash.update(fs.readFileSync(filePath));
-        fs.renameSync(filePath, filePath.replace('.js', '.classic.js'));
+
+        if (fileName !== 'unstable_server-external-runtime.js') {
+          fs.renameSync(filePath, filePath.replace('.js', '.classic.js'));
+        }
```

Comment:
We don't have access to bundle types at this point. Instead of relying on `fileName` I can parse bundles array and try finding everything what has bundleType `FB_WWW_BROWSER_SCRIPT`, if that seems like a better option